components/phoenix/combicon-mc.stanza

```diff
@@ -0,0 +1,63 @@
+#use-added-syntax(jitx)
+defpackage ocdb/phoenix/combicon-mc :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+public pcb-component component (n:Int):
+  name = "Phoenix screw terminal"
+  manufacturer = "Phoenix"
+  mpn = switch(n) :
+    2 : "1803277"
+    3 : "1803280"
+    4 : "1803293"
+    5 : "1803303"
+    6 : "1803316"
+    7 : "1803329"
+    8 : "1803332"
+    9 : "1803345"
+    10 : "1803358"
+    11 : "1803361"
+    12 : "1803374"
+    13 : "1803387"
+    14 : "1803390"
+    15 : "1803400"
+    16 : "1803413"
+    18 : "1841297"
+    20 : "1841271"
```

Comment:
why not having a map from `Int` to `String` : [2, "1803277"], ...?

Reply:
I think the code here is pretty clear. It's not efficient, but that doesn't matter in this context.

Reply:
`public pcb-component component (n:Int):` could be much shorter if you move outside the mappings